### PR TITLE
notifications for teams changing

### PR DESCRIPTION
**Pull request by maxtaco**
- still some todos in terms of busting local caches (cc: @miles)
- needs some server support from keybase/keybase#1464 (cc: @modalduality)
- propagate through to KBFS or whoever else is listening
- works with existing tests, but could benefit from a test of its own

### CODE REVIEW

File: go/systests/teams_test.go [hunk 1]
```diff
@@ -45,8 +45,14 @@ func TestTeamRotateOnRevoke(t *testing.T) {
 	}
 	secretBefore := before.Data.PerTeamKeySeeds[before.Generation()].Seed.ToBytes()
 
+	// User1 sould get a gregor that the team he was just added to changed.
+	tt.users[1].waitForTeamChagnedGregor(team, keybase1.Seqno(2))
```
Comment (patrickxb) [inline go/systests/teams_test.go]: spelling

**Comment by maxtaco** [comment]
Thanks, I need @modalduality to help with keybase/keybase#1464 before I can land

**Comment by mmaxim** [comment]
I believe @patrickxb was talking about `waitForTeamChagnedGregor` for the spelling fix (just randomly popped in here)

**Comment by patrickxb** [comment]
Yes, that's what I was referring to.

**Comment by maxtaco** [comment]
thanks, sorry i misunderstood!

**Comment by keybase-ci-visdiff** [comment]
The commits 8290121b8497...185eadfa658a introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1499445179486-8290121b8497-185eadfa658a-linux/Markdown-emoji-8290121b8497.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1499445179486-8290121b8497-185eadfa658a-linux/Markdown-emoji-185eadfa658a.png)  
   Markdown-emoji comparison
</summary>

**Pull request merged**